ibis/backends/pandas/executor.py

```diff
@@ -167,6 +167,8 @@ def visit(cls, op: ops.IsNan, arg):
     def visit(
         cls, op: ops.SearchedCase | ops.SimpleCase, cases, results, default, base=None
     ):
+        if not cases:
+            return default
```

Comment:
This is a weird edge case. We should enforce through our pattern system that `cases` is never empty.

Reply:
now that we officially disallowed empty cases in https://github.com/ibis-project/ibis/pull/9559, this is no longer needed.